src/plugins/data/public/ui/search_bar/search_bar.tsx

```diff
@@ -365,6 +367,7 @@ class SearchBarUI extends Component<SearchBarProps, State> {
           onSubmit={this.onQueryBarSubmit}
           indexPatterns={this.props.indexPatterns}
           isLoading={this.props.isLoading}
+          isFill={this.props.fillSubmitButton || false}
```

Comment:
I think we should also have it as `fillSubmitButton` on this level. Sorry I wasn't clear about this before

Reply:
done b3fe5b82f60b78857a55e6dcfda47a44eda885ab

Reply:
@Dosant I should have done it anyway. It's good to have the same prop name from source to dest.